docs/index.md

```diff
@@ -40,6 +40,16 @@ You can test this config by creating a `test.tf` and run terraform commands from
 - Build the infrastructure: `terraform apply`
 
 ```hcl
+variable "project_id" {
```

Comment:
Could we add this content to a dedicated guide? It is starting to be a very big example that is not practical to read.